src/app/components/pages/info-page/info-page.component.html

```diff
@@ -1,67 +1,298 @@
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
+          <b>Joliette</b><br/>
+          Carmen Bruda<br/>
+          joliette@nousrire.com<br/>
+          514-570-3443
+        </div>
+      </div>
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
+          <b>Kamouraska</b><br/>
+          Cindy Garneau<br/>
+          kamouraska@nousrire.com<br/>
+          418-551-4802
+        </div>
+      </div>
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
+          <b>Laval</b><br/>
+          Carmen Bruda<br/>
+          laval@nousrire.com<br/>
+          514-570-3443
+        </div>
+      </div>
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
+          <b>Mascouche/Terrebonne</b><br/>
+          Carmen Bruda<br/>
+          mascouche-terrebonne@nousrire.com<br/>
+          514-570-3443
+        </div>
+      </div>
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
+          <b>Montréal</b><br/>
+          Amélie Vadeboncoeur<br/>
+          benevoles@nousrire.com<br/>
+          514-817-0728
+        </div>
+      </div>
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
+          <b>Québec</b><br/>
+          Maude Lavoie<br/>
+          quebec@nousrire.com<br/>
+          418-576-5172
+        </div>
+      </div>
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
+          <b>Sherbrooke</b><br/>
+          Marie-Laurence Marcoux<br/>
+          sherbrooke@nousrire.com<br/>
+          819-575-2276
+        </div>
+      </div>
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
+          <b>St-Hubert</b><br/>
+          Marie-Ève Bédard<br/>
+          st-hubert@nousrire.com<br/>
+          514-531-5186<br/>
+        </div>
+      </div>
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
+          <b>Sutton</b><br/>
+          Jessica Martin<br/>
+          sutton@nousrire.com<br/>
+          579-436-8255
+        </div>
+      </div>
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
+          <b>Val-David</b><br/>
+          Carmen Bruda<br/>
+          val-david@nousrire.com<br/>
+          514-570-3443
+        </div>
+      </div>
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
+          <b>Victoriaville</b><br/>
+          Caroline Mercier<br/>
+          victoriaville@nousrire.com<br/>
+          819-990-1886
+        </div>
+      </div>
+      <div class="row cell-box-row">
+        <div class="col-md-12 cell-box">
+          <b>Ville Saguenay</b><br/>
+          Caroline Laflamme<br/>
+          villesaguenay@nousrire.com<br/>
+          514-771-5496
+        </div>
+      </div>
+    </div>
+
+    <h3 class="info-page__title">
+      <b><u>Description des différentes missions proposées</u></b>
```

Comment:
We don't use `<b>`, `<u>`, `<i>` and other tags of this kind since the design is in the CSS only.